src/main/java/com/jcabi/log/Logger.java

```diff
@@ -368,6 +388,16 @@ public void write(final int data) throws IOException {
         } else {
             logger = LoggerFactory.getLogger(source.getClass());
         }
+        try {
+            final Field fqcn = logger.getClass()
+                .getDeclaredField("FQCN");
+            setFinalStatic(fqcn, Logger.class.getName());
+            // @checkstyle EmptyBlockCheck (5 lines)
+        } catch (final NoSuchFieldException ex) {
+        } catch (final SecurityException ex) {
+        } catch (final IllegalArgumentException ex) {
```

Comment:
@dmzaytsev @yegor256 

You just can reject the entire PR. We intentional catch and ignore these exception for non-log4j binding.